nautobot/ipam/tests/test_filters.py

```diff
@@ -1249,3 +1254,278 @@ def test_virtual_machine(self):
+        self.assertEqual(Prefix.objects.filter(network__net_equals="192.1.0.0/32").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_equals="fd78:da4f:e596:c217::/64").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_equals="fd78:da4f:e596:c218::/64").count(), 0)
+
+    def test_net_contained(self):
+        self.assertEqual(Prefix.objects.filter(network__net_contained="192.0.0.0/8").count(), 7)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="192.168.0.0/16").count(), 6)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="192.168.3.0/24").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="192.168.1.0/24").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="192.168.3.192/28").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="192.168.3.192/32").count(), 0)
+
+        self.assertEqual(Prefix.objects.filter(network__net_contained="fd78:da4f:e596:c217::/64").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="fd78:da4f:e596:c217::/120").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contained="fd78:da4f:e596:c218::/64").count(), 0)
+
+    def test_net_contained_or_equal(self):
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="192.0.0.0/8").count(), 7)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="192.168.0.0/16").count(), 7)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="192.168.3.0/24").count(), 4)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="192.168.1.0/24").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="192.168.3.192/28").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="192.168.3.192/32").count(), 0)
+
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="fd78:da4f:e596:c217::/64").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="fd78:da4f:e596:c217::/120").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="fd78:da4f:e596:c218::/64").count(), 0)
+
+    def test_net_contains(self):
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.0.0.0/8").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.0.0/16").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.0/24").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.192/28").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.192/30").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.192/32").count(), 3)
```

Comment:
Should this be 4? (192.168.0.0/16, 192.168.3.0/24, 192.168.3.192/28, 192.168.3.192/30)

Reply:
`192.168.3.192/30` is being queried, but it is never created, only the /28